src/main/java/bsh/engine/BshScriptEngine.java

```diff
@@ -362,14 +360,14 @@ public Object invokeFunction(String name, Object... args) throws ScriptException
         Interpreter.
     */
 
-    class WriterOutputStream extends OutputStream {
+    static public class WriterOutputStream extends OutputStream {
 
```

Comment:
I was trying to use this in my test, but I could not figure it out... I will come back on this if the rest is ok.